base.yaml

```diff
@@ -32,6 +50,45 @@ sensor:
     name: "Openings"
     unit_of_measurement: "openings"
     icon: mdi:open-in-app
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_total
+    type: paired_devices_total
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Devices"
+    icon: mdi:remote   
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_remotes
+    type: paired_devices_remotes
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Remotes"
+    disabled_by_default: true
+    icon: mdi:remote  
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_keypads
+    type: paired_devices_keypads
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Keypads"
+    disabled_by_default: true
+    icon: mdi:dialpad  
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_wall_controls
+    type: paired_devices_wall_controls
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Wall Controls"
+    disabled_by_default: true
+    icon: mdi:pan-vertical
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_accessories
+    type: paired_devices_accessories
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Accessories"
+    disabled_by_default: true
+    icon: mdi:cloud-cancel-outline  
```

Comment:
@bdraco My guess is that a lot of memory usage comes from these extra components, can you check how it looks if you comment out the ones with `disabled_by_default: true`

Reply:
👍  Will give that a try after I finish debugging the button

Reply:
Taking them out solves the memory issue

I'm resuming testing